lib/mutant/integration/rspec.rb

```diff
@@ -119,12 +139,6 @@ def all_examples
         end
       end
 
-      def filter_examples(&predicate)
-        @rspec_world.filtered_examples.each_value do |examples|
```

Comment:
Per mutation this was for 99% of the examples resulting in the same result: Clearing the array entirely.